geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/collections/SizeableObject2ObjectOpenCustomHashMapWithCursor.java

```diff
@@ -226,6 +226,14 @@ int calculateBackingArraysOverhead() {
         + BACKING_ARRAY_LENGTH_COEFFICIENT * getTotalBackingArrayLength();
   }
 
+  private <E> int getElementSize(E element) {
+    if (element instanceof Sizeable) {
```

Comment:
could this class also use your new SizeableObjectSizer?

Reply:
Yes, good catch, thanks